runtime/parachains/src/disputes.rs

```diff
@@ -1313,6 +1313,10 @@ fn check_signature(
 	}
 }
 
+#[cfg(test)]
+#[allow(unused_imports)]
```

Comment:
why is it needed? isn't it used when cfg(test) is enabled?

Reply:
I saw an error in CI about unused imports, hence I added it. This will be refactored soon after #4834 since there is quite a bit of duplicate test code.